### PR TITLE
Add July main deck, and June YouTube playlist

### PR DESCRIPTION
**Pull request by myitcv**

### CODE REVIEW

File: meetups/README.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
+🗣 Beth Anderson - Go: An Origin Story (to follow)
+
+🗣 Nick Wells - Semantic versioning - what Go programmers need to know (to follow)
+
+🗣 Daniela Petruzalek - Pacman from scratch (to follow)
+
+### June 19 2019 - [Monzo](https://monzo.com)
+
+🎥 [YouTube Playlist](https://www.youtube.com/playlist?list=PL8QGElREVyDCmYhUr65OCtWQRdltb-Khs)
```
Comment (thoeni) [inline meetups/README.md]: This will be updated asap... still to be created

Comment (thoeni) [reply meetups/README.md]: `https://www.youtube.com/playlist?list=PL8QGElREVyDDD0ZMYGJoq3YnQeDzUzwtX`

File: meetups/README.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+
+🗣 [Beth Anderson - Go: An Origin Story](https://docs.google.com/presentation/d/e/2PACX-1vT7mcAj97QKVuLIZpG4-n6P6ozEmuuF8CclGQvTyJrgYBGTR5BuRsjdLng66JjIpSvFxmvMlL3ayNiw/pub)
+
+🗣 [Nick Wells - Semantic versioning - what Go programmers need to know](https://talks.godoc.org/github.com/nickwells/presentations/semver.slide#1)
+
+🗣 [Daniela Petruzalek - Pacman from scratch (to follow)](https://docs.google.com/presentation/d/e/2PACX-1vSxfpqZerJXYoPwtArNsQziPxpgCq5SvdSNcG4Z3UlxOXDRNtMLergbUJRfSSrZawd--Wv5ZxgaAGhf/pub)
+
+### June 19 2019 - [Monzo](https://monzo.com)
+
+🎥 [YouTube Playlist](https://www.youtube.com/playlist?list=PL8QGElREVyDCmYhUr65OCtWQRdltb-Khs)
```
Comment (thoeni) [inline meetups/README.md]:
```suggestion
🎥 [YouTube Playlist](https://www.youtube.com/playlist?list=PL8QGElREVyDDD0ZMYGJoq3YnQeDzUzwtX)
```

Comment (myitcv) [reply meetups/README.md]: Done

**Comment by myitcv** [comment]
All done @thoeni

**Pull request merged**